### PR TITLE
feat(): Added enterprise option

### PR DESCRIPTION
**Pull request by roshanikk**

### CODE REVIEW

File: cmd/install.go [hunk 1]
```diff
@@ -7,8 +7,9 @@ import (
 )
 
 var (
-	profile   string
-	skipSteps = []string{}
+	profile    string
+	skipSteps  = []string{}
+	enterprise bool
```
Comment (that-backend-guy) [inline cmd/install.go:12]: do we need the `enterprise bool` flag? We can directly install kubeslice-ui if the chart configuration was provided by the end user. this flag seems redundant

**Review by that-backend-guy — CHANGES_REQUESTED** [review]
enterprise flag is redundant and should be removed

File: cmd/install.go (hunk 1 above)
Comment (roshanikk) [reply cmd/install.go:12]: Agreed

File: cmd/install.go [hunk 2]
```diff
@@ -71,4 +71,5 @@ Supported values:
 	- worker-registration: Skips the registration of KubeSlice Workers on the Controller
 	- worker: Skips the installation of KubeSlice Worker
 	- demo: Skips the installation of additional example applications`)
```
Comment (priyank-upadhyay) [inline cmd/install.go:73]: We can add a skip option for `ui` as well.

**Review by priyank-upadhyay — APPROVED** [review]

**Pull request merged**